src/Shell/ServerShell.php

```diff
@@ -100,6 +111,12 @@ public function startup()
         if (preg_match("/^([a-z]:)[\\\]+(.+)$/i", $this->_documentRoot, $m)) {
             $this->_documentRoot = $m[1] . '\\' . $m[2];
         }
+        if (substr($this->_iniPath, -1, 1) === DIRECTORY_SEPARATOR) {
```

Comment:
`$this->_iniPath = rtrim($this->_iniPath, DIRECTORY_SEPARATOR)` would be simpler.

Reply:
Ditto.